jbmestar/index.html

```diff
@@ -0,0 +1,207 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8">
+    <style>
+        .ul {
+            list-style-type: none;
+            margin: 0;
+            padding: 0;
+            overflow: hidden;
+            background-color: #333;
+        }
+
+        .li {
+            float: left;
+        }
+
+        .li a {
+            display: block;
+            color: white;
+            text-align: center;
+            padding: 14px 16px;
+            text-decoration: none;
+        }
+
+        .li a:hover {
+            background-color: #111;
+        }
+
+        table {
+            margin-bottom: 5%;
+        }
+    </style>
+    <link href="../../src/stylesheets/bootstrap.min.css" rel="stylesheet" type="text/css">
+    <link href="../../src/stylesheets/style.css" rel="stylesheet" type="text/css">
+    <script async="" src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
+    <meta name="viewport" src="width=device-width,%20initial-scale=1,%20shrink-to-fit=no">
+    <title>iOS Jailbreak Guide - Example Guide</title>
+    <script src="../../src/js/jquery.min.js"></script>
+    <link rel="shortcut icon" href="https://i.imgur.com/XoJGlxQ.png" type="image/x-icon">
+    <link rel="icon" href="https://i.imgur.com/XoJGlxQ.png" type="image/x-icon">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css"
+
+      integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU"
+
+      crossorigin="anonymous">
+    <script src="../../src/js/bootstrap.bundle.min.js"></script>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css"
+
+      integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU"
+
+      crossorigin="anonymous">
+    <script src="../../src/js/scrollToTop.js"></script>
+    <link href="../../src/stylesheets/bootstrap.min.css" rel="stylesheet" type="text/css">
+    <link href="../../src/stylesheets/style.css" rel="stylesheet" type="text/css">
+    <meta name="site-name" content="Jailbreak Guide - Example Guide">
```

Comment:
```suggestion
    <meta name="site-name" content="Jailbreak Guide - Star">
```

Replace the example placeholder stuff

Reply:
Keep in mind this isn't the only occurrence that needs to be changed.